src/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --button-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 15px 5px 45px;
+}
+
+.card__name {
+  margin-bottom: 6px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  margin-bottom: 15px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15px;
+  margin-left: -4px;
+}
+
+.card__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__star_active {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+
+.price__value {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+  color: #fff;
+  background-color: var(--button-color);
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  color: var(--button-color);
+  background-color: #fff;
+  border: 1px solid var(--button-color);
```

Comment:
at this moment when you hovered on your link - the border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change its position, so it's better if you write border property inside the button.